.storybook/addons/codeEditorAddon/editor.js

```diff
@@ -205,6 +205,11 @@ export class EditorElement extends LitElement {
     }
   }
 
+  generateStackblitzProject() {
```

Comment:
Let's make this an arrow function
```suggestion
  const generateStackblitzProject = () => {
```

Reply:
The 'const' keyword leads to build issues because it's being used to declare a class member